climada/engine/test/test_impact.py

```diff
@@ -171,7 +176,6 @@ def test_impact_per_year_sum(self):
         imp.at_event[7] = 0.381063674256423e9
         imp.at_event[8] = 0.569142464157450e9
         imp.at_event[9] = 0.467572545849132e9
-        imp.unit = 'USD'
```

Comment:
What is the reason to remove the unit here? Is it not needed?

Reply:
exactly.